chain/network/src/routing_table_actor.rs

```diff
@@ -50,28 +385,44 @@ impl Handler<StopMsg> for RoutingTableActor {
     }
 }
 
-impl Actor for RoutingTableActor {
-    type Context = SyncContext<Self>;
-}
-
+// Messages for RoutingTableActor
 #[derive(Debug)]
 pub enum RoutingTableMessages {
+    // Add verified edges to routing table actor (Will be removed soon).
     AddEdges(Vec<Edge>),
+    // Remove edges. (Will be removed soon).
     RemoveEdges(Vec<Edge>),
+    // Add verified edges to routing table actor and update stats.
```

Comment:
explain what 'verified' means

Reply:
// Add verified edges to routing table actor and update stats.
    // Each edge contains signature of both peers to which given edge belongs to.
    // We say that the edge is "verified" if and only if we checked that the `signature0` and
    // `signature1` is valid.

Reply:
// Add verified edges to routing table actor and update stats.
    // Each edge contains signature of both peers.
    // We say that the edge is "verified" if and only if we checked that the `signature0` and
    // `signature1` is valid.